### ISSUE TITLE
For public URL its connecting perfectly but not with protected URL with username and password

### ISSUE THREAD

**Issue by maan-smith** [ant-media/LibRtmp-Client-for-Android#46]
Hi for open link the app is running perfect but not for closed link. I have tested that in iOS and its running smooth but not on android can you let me know what I am doing wrong or I need to configure.
rtmp://{username}:{password}@p.ep529710.i.akamaientrypoint.net/EntryPoint/{streaming_id}

rtmp://352769:360301@p.ep529710.i.akamaientrypoint.net/EntryPoint/dclive_1_150@529710

**Comment by mekya** [comment]
Hi @maan-smith ,

It seems that librtmp parse_url function does not support "user_name" and "password". It seems that this functionality should be added to librtmp in order to meet your request. 

Is it possible for you adding this feature and making a PR?

**Comment by maan-smith** [comment]
No, I had no idea about that can you please do that or someone you know to
do that?

**Comment by mekya** [comment]
yeah we can add this feature to the lib however we are busy at these times. 

when do you need this functionality?

**Comment by maan-smith** [comment]
Take your time I need after 4 days

**Comment by mekya** [comment]
I mean, It would take a couple of weeks to start implementing this feature.

**Comment by maan-smith** [comment]
No issue just do that. I am waiting for positive feedback
